<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vent_menu</class>
 <widget class="QMainWindow" name="vent_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>MENU PRINCIPAL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clientes">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CLIENTES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_empleados">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>EMPLEADOS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clientesEmpresa">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CLIENTE EMPRESA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_detalleVenta">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>DETALLE DE VENTA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_productos">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PRODUCTOS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cerrar">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CERRAR VENTANA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_documentoVenta">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>DOCUMENTO DE VENTAS</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>439</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
